src/Makefile

```diff
@@ -0,0 +1,21 @@
+PROG=		checkrc
+SRCS=		main.c \
+		validation_utils.c
```

Comment:
maybe it would help to add include/config.h as dependency, too, since it is

Reply:
There's no need, the framework generates and includes dependency files (found in `obj/.depend*`) automatically, listing all headers found by the compiler :)